pkgs/development/python-modules/laspy/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, laszip
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "laspy";
+  version = "2.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-Wdbp6kjuZkJh+pp9OVczdsRNgn41/Tdt7nGFvewcQ1w=";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    laszip
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ pname "laszip" ];
```

```suggestion
  pythonImportsCheck = [ "laspy" "laszip" ];
```
we are not using pname here because logical in python those things are not the same